sql3/planner/inbuiltfunctionsstring.go

```diff
@@ -644,3 +668,65 @@ func (n *callPlanExpression) EvaluateReplicate(currentRow []interface{}) (interf
 	return strings.Repeat(stringArg, int(intArg)), nil
 
 }
+
+// function to get the index of substring in the given string
```

Comment:
The exported functions should follow this format
``` EvalutateCharIndex ... [and then describe what it does and any information that would be useful for a user or another developer to know] ```
More info about that here: https://go.dev/blog/godoc

Reply:
Updated